LPC Projects/PWM_Sample/src/PWM_Sample.c

```diff
@@ -62,18 +227,83 @@ int main(void)
 	LPC_TIMER16_0->PWMC = 0x1;
 	Chip_TIMER_Enable(LPC_TIMER16_0);
 
+	Chip_TIMER_Init(LPC_TIMER16_1);
+	Chip_TIMER_Reset(LPC_TIMER16_1);
+	Chip_TIMER_MatchEnableInt(LPC_TIMER16_1, 0);
+	Chip_TIMER_SetMatch(LPC_TIMER16_1, 0, 10000);
+	Chip_TIMER_ResetOnMatchEnable(LPC_TIMER16_1, 0);
+	LPC_TIMER16_1->PR = 48;
+	Chip_TIMER_Enable(LPC_TIMER16_1);
+	NVIC_ClearPendingIRQ(TIMER_16_1_IRQn);
+	NVIC_EnableIRQ(TIMER_16_1_IRQn);
+
+    // Uart setup
+	Chip_IOCON_PinMuxSet(LPC_IOCON, IOCON_PIO1_6, (IOCON_FUNC1 | IOCON_MODE_INACT));/* RXD */
+	Chip_IOCON_PinMuxSet(LPC_IOCON, IOCON_PIO1_7, (IOCON_FUNC1 | IOCON_MODE_INACT));/* TXD */
+	Chip_UART_Init(LPC_USART);
+	Chip_UART_SetBaud(LPC_USART, 115200);
+	Chip_UART_ConfigData(LPC_USART, (UART_LCR_WLEN8 | UART_LCR_SBS_1BIT));
+	Chip_UART_SetupFIFOS(LPC_USART, (UART_FCR_FIFO_EN | UART_FCR_TRG_LEV2));
+	Chip_UART_TXEnable(LPC_USART);
+
+	// CAN
+	LPC_CCAN_API->init_can(&CanApiClkInitTable[0], TRUE);
+	/* Configure the CAN callback functions */
+	LPC_CCAN_API->config_calb(&callbacks);
+	/* Enable the CAN Interrupt */
+	NVIC_EnableIRQ(CAN_IRQn);
+	NVIC_ClearPendingIRQ(CAN_IRQn);
+
+	msg_obj.msgobj = 0;
+	msg_obj.mode_id = 0x000;
+	msg_obj.mask = 0x000;
+	LPC_CCAN_API->config_rxmsgobj(&msg_obj);
+
+
+    /* Configure GPIO pins as input pins */
+    Chip_GPIO_SetPinDIRInput(LPC_GPIO, PIN_A_PORT, PIN_A);
+    Chip_GPIO_SetPinDIRInput(LPC_GPIO, PIN_B_PORT, PIN_B);
```

Comment:
Being nitpicky here, but the spacing on these three lines is different from the rest of the file